src/opower/utilities/aepbase.py

```diff
@@ -112,6 +117,12 @@ async def async_login(
         optional_mfa_secret: Optional[str],
     ) -> None:
         """Login in AEP using user/pass and then do the SAML call to opower."""
+        # Clear cookies before logging in again, in case old ones are still around
+        if cls.login_cookies_to_clear() is not None:
```

Comment:
Revert all other changes (except removal of `if not login_parser.password_field_found:` block) and just call here:
`session.cookie_jar.clear(lambda cookie: cookie["domain"].endswith(".opower.com"))`

Reply:
Done, makes it simple, thanks